mkosi

```diff
@@ -3240,8 +3291,8 @@ def load_args() -> CommandLineArguments:
             args.boot_protocols = ["uefi"]
         if not {"uefi", "bios"}.issuperset(args.boot_protocols):
             die("Not a valid boot protocol")
-        if "bios" in args.boot_protocols:
-            warn("bios boot not implemented yet")
+        if "bios" in args.boot_protocols and args.distribution not in (Distribution.fedora,):
+            die("bios boot not implemented yet for %s" % args.distribution)
```

Comment:
Please use an f-string here too. %s is too ugly.

Reply:
I forgot about this one, will do.